apps/application/views.py

```diff
@@ -1,7 +1,7 @@
 import csv
 import uuid
 from collections import OrderedDict
-from slacker import Slacker
+from utils.slack import send_to_slack
```

Comment:
pro tip: we like to organize imports as follows
```
import [included python packages]
from [included python packages] import ____

import [rest framework packages]
...

import [django packages]
...

import [madras packages]
...

import [utils from madras]
...
```

so put the utils import at the bottom of the imports